deploy/crds/performance.openshift.io_v1alpha1_performanceprofile_cr.yaml

```diff
@@ -2,6 +2,18 @@ apiVersion: performance.openshift.io/v1alpha1
 kind: PerformanceProfile
 metadata:
   name: example-performanceprofile
+  namespace: openshift-performance-addon
 spec:
-  # Add fields here
-  size: 3
+  cpu:
+    isolated: "2-3"
+    nonIsolated: "0"
```

Comment:
Could you please clarify the state of CPU 1? how could it be neither `isolated` nor `nonIsolated`?

Reply:
maybe even add a validation hook on that later on ?

Reply:
it is just example that will be injected under the CSV and does not the real case
regarding validation webhook I left `TODO` - https://github.com/openshift-kni/performance-addon-operators/blob/fd9054e83fef41af7f7683af9255a452dd7bd09e/pkg/controller/performanceprofile/performanceprofile_controller.go#L229

Reply:
Ack, good enough for me now

Reply:
@slintes this CR example injected into CSV, so I do not add some comments to it(unsure regarding parsing), what example do you prefer?